applications/nrf_desktop/README.rst

```diff
@@ -1380,7 +1380,7 @@ The nRF Desktop devices use one of the following Link Layers:
     * When :kconfig:`CONFIG_CAF_BLE_USE_LLPM` is disabled, the device will use only standard Bluetooth LE connection parameters with the lowest available connection interval of 7.5 ms.
 
       If the LLPM is disabled and more than 2 simultaneous Bluetooth connections are supported (:kconfig:`CONFIG_BT_MAX_CONN`), you can set the value for :kconfig:`CONFIG_SDC_MAX_CONN_EVENT_LEN_DEFAULT` to ``2500``.
-      With this value, the nRF Desktop central is able to exchange the data with up to 3 Bluetooth LE peripherals during every 7.5-ms connection interval.
+      With this value, the nRF Desktop central is able to exchange the data with up to 3 Bluetooth LE peripherals during every 7.5 ms connection interval.
```

Comment:
We do use hyphens for modifiers like this. Please revert globally.

Reply:
Here I agree the hyphened version is appropriate.

Reply:
Okay, will fix throughout where it is applicable/used as modifier.